config/fcos/v1_5_exp/translate.go

```diff
@@ -362,3 +362,28 @@ func buildGrubConfig(gb Grub) string {
 	superUserCmd := fmt.Sprintf("set superusers=\"%s\"\n", strings.Join(allUsers, " "))
 	return "# Generated by Butane\n\n" + superUserCmd + strings.Join(cmds, "\n") + "\n"
 }
+
+func arePartitionEqual(p1, p2 types.Partition) bool {
+    return p1.Label == p2.Label && p1.Number == p2.Number
+}
+
+
+func checkPartitionTable(disk string, wipeTable bool, partitions []types.Partition) error {
```

Comment:
This function isn't actually called anywhere?

Validation checks should be done in the validation pass if at all possible.  We have an unusual case here though: this is a validation that runs solely on structs defined in `base`, but the validation needs to be in `config/fcos`.  (Non-FCOS-derived distros don't have `/dev/disk/by-id/coreos-boot-disk` and so configs for those distros don't have a way to denote the boot disk.)  `config/fcos` can't define a `Validate()` method on `Disk` because that struct is defined in `base`, so we'll need to define `Validate()` on the top-level `Config` struct in `config/fcos` instead.